splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -125,11 +128,15 @@ private String detectRumVersion() {
         return "unknown";
     }
 
-    private static void recordInitializationSpan(long startTimeNanos, List<InitializationEvent> initializationEvents, Tracer tracer, Config config) {
+    private void recordInitializationSpans(long startTimeNanos, List<InitializationEvent> initializationEvents, Tracer tracer, Config config) {
+        Span overallAppStart = startupTimer.start(tracer);
+        System.out.println("overallAppStart = " + overallAppStart);
```

Comment:
```suggestion
```

Reply:
I think I'm juggling too many things at once. Thanks for keeping me honest on this stuff!